shell/platform/windows/win32_window_unittests.cc

```diff
@@ -1,22 +1,30 @@
-#include "flutter/shell/platform/windows/testing/win32_flutter_window_test.h"
+#include "flutter/shell/platform/windows/testing/win32_window_test.h"
 #include "gtest/gtest.h"
 
 namespace flutter {
 namespace testing {
 
-TEST(Win32FlutterWindowTest, CreateDestroy) {
-  Win32FlutterWindowTest window(800, 600);
+TEST(Win32WindowTest, CreateDestroy) {
+  Win32WindowTest window;
   ASSERT_TRUE(TRUE);
 }
 
-TEST(Win32FlutterWindowTest, CanFontChange) {
-  Win32FlutterWindowTest window(800, 600);
+TEST(Win32WindowTest, CanFontChange) {
+  Win32WindowTest window;
+  window.InitializeChild("FLUTTERVIEW", 10, 10);
+
   HWND hwnd = window.GetWindowHandle();
   LRESULT result = SendMessage(hwnd, WM_FONTCHANGE, NULL, NULL);
   ASSERT_EQ(result, 0);
   ASSERT_TRUE(window.OnFontChangeWasCalled());
   ASSERT_TRUE(TRUE);
+
+  window.Destroy();
```

Comment:
Why are you adding the explicit call? It should be destroyed when it goes out of scope.

Reply:
That's what I thought, but without this line, the test seems to hang and fail. I'll take a closer look into why that is.